src/utils.js

```diff
@@ -14,12 +14,9 @@ const { URL } = require('url');
 const fetchAPI = require('@adobe/helix-fetch');
 
 function createFetchContext() {
-  /* istanbul ignore next */
-  if (process.env.HELIX_FETCH_FORCE_HTTP1) {
```

Comment:
I would keep the `if` and just set `process.env.HELIX_FETCH_FORCE_HTTP1 = 'true'`